src/verifier_contract/src/main.nr

```diff
@@ -0,0 +1,75 @@
+use dep::aztec::macros::aztec;
+
+// Verifier implementation with Merkle proof verification
+#[aztec]
+pub contract Verifier {
```

Comment:
let's call it Authorizer (?)
or MerkleInclussionAuthorizer ?

we could make a folder with diff implementations of the "interface" for different mechanisms